extensions/sql-migration/src/scripts/SHIR-auto-configuration.ps1

```diff
@@ -559,7 +559,7 @@ Function Register-IntegrationRuntime {
 			Write-OutputAndLog "Installing Az.DataMigration Module..."
 			$stopwatch = New-Object System.Diagnostics.Stopwatch
 			$stopwatch.Start()
-			Install-Module -Name Az.DataMigration -Force
+			Install-Module -Name Az.DataMigration -Force -AllowClobber
```

Comment:
This allows overrating installation files even if they are protected or marked as read-only.   

Is this behavior expected? i would say check with PM.